administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-06-08.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS `#__cookiemanager_groups` (
+  `id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(64) NOT NULL,
+  `params` varchar(1024) NOT NULL DEFAULT ''
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE utf8mb4_unicode_ci;
+
+INSERT IGNORE INTO `#__cookiemanager_groups` (`title`) VALUES
+('Cookie Consent Banner'),
+('Cookie Settings Banner'),
+('Cookie Consent Information');
```

Comment:
```suggestion
INSERT INTO `#__cookiemanager_groups` (`id`, `title`, `params`) VALUES
(1, 'Cookie Consent Banner', '{}'),
(2, 'Cookie Settings Banner', '{}'),
(3, 'Cookie Consent Information', '{}');
```
As this is a new table which we create with the update from 3.10 or previous J4 versions in this SQL script, we don't have to assume any present records from before the update, so we
- can use hard-coded ID's
- can initialize the `params` column with an empty json (without this it would use the empty string default value of that column)
- don't need `IGNORE` (you only need this if you run the statement or the complete script twice by accident, but we can assume this doesn't happen when updating)
similar to what we do in other update SQL scripts for adding new extensions.

Reply:
Later in the development cycle when we know which params we have, we can use the right jason instead of `'{}'` in these statements.

Reply:
Thanks